### ISSUE TITLE
Skip to content button is not accessible in Safari

### ISSUE THREAD

**Issue by FlyersPh9** [iTwin/iTwinUI#512]
Safari doesn't support `:focus-visible` and I think that is making the element un-focusable.

**Comment by mayank99** [comment]
Can you try replacing this line
https://github.com/iTwin/iTwinUI/blob/e40f387d94781fb0dcb6ebce0bbfea74c4404931/src/skip-to-content/skip-to-content.scss#L7
with this?
```css
outline: 2px solid var(--iui-color-foreground-primary);
outline-offset: 2px;
```

Even if that works, I think we might want to investigate why this fallback is not working.
https://github.com/iTwin/iTwinUI/blob/e40f387d94781fb0dcb6ebce0bbfea74c4404931/src/style/mixins.scss#L28-L32

**Comment by FlyersPh9** [comment]
I followed the steps on [this page](https://stackoverflow.com/q/52783375) to enable tabbing in Safari, no action needed.
